etc/cvmfs/config.d/ligo.osgstorage.org.conf

```diff
@@ -1,4 +1,6 @@
-CVMFS_EXTERNAL_URL="https://stashcache.t2.ucsd.edu:8444//user/ligo/;https://osg-kansas-city-stashcache.nrp.internet2.edu:8444//user/ligo/;https://osg-chicago-stashcache.nrp.internet2.edu:8444//user/ligo/;https://osg-new-york-stashcache.nrp.internet2.edu:8444//user/ligo/;https://dtn2-daejeon.kreonet.net:8444//user/ligo/"
+# subset of stashcache servers configured by OSG
+CVMFS_EXTERNAL_URL="https://stashcache.t2.ucsd.edu:8444//user/ligo/;https://osg-kansas-city-stashcache.nrp.internet2.edu:8444//user/ligo/;https://osg-chicago-stashcache.nrp.internet2.edu:8444//user/ligo/;https://osg-new-york-stashcache.nrp.internet2.edu:8444//user/ligo/;https://dtn2-daejeon.kreonet.net:8444//user/ligo/;https://ligo.hpc.swin.edu.au:8443//user/ligo/"
```

Comment:
Just wondering, are the double slashes after the port numbers typos?

Reply:
Probably.  I think the first one was set that way and everybody has just been following the same conventions since when they add a new one.

Reply:
But in this case isn't it interpreted by cvmfs instead of xrootd? I guess it would still turn into an http GET //... on the xrootd server.